index.bs

```diff
@@ -2497,12 +2503,17 @@ the credential. It is intended primarily for [RPS] that wish to tightly control
     the available authenticators match a provided AAGUID, the client MUST select an authenticator from among the available
     authenticators to generate the credential.
 
+: Client data
+:: Returns the JSON value `true` to indicate to the RP that the extension was acted upon
+
 : Authenticator argument
-:: There is no authenticator argument.
+:: none
 
 : Authenticator processing
-:: None.
+:: none
```

Comment:
Editorial nit: Why omit capitalization and punctuation in this one item when others are capitalized and punctuated?